applications/tari_miner/src/stratum/controller.rs

```diff
@@ -243,6 +243,7 @@ impl Controller {
             // problem with template
             let _result = self.send_message_get_job_template();
         } else {
+            // unhandled
```

Comment:
It only kinda bugs me that the comments aren't consistent 🙃 

What's the need for them all of a sudden though, all they failing a linter now or something?

Reply:
yip cant have an empty else { }
tried to make them as far as I could see comment why its empty

Reply:
You can, but having an actual comment in there is more useful than a macro bypassing this check.